### ISSUE TITLE
Code from hero image does not work

### ISSUE THREAD

**Issue by poke** [microsoft/just#153]
The code from the hero image on [microsoft.github.io/just](https://microsoft.github.io/just/)—[here in the source](https://github.com/microsoft/just/blob/857fd9a6cea82c8ae564b7f4f65fb8c18d6b579b/docs/img/sample.png)—will not work.

The code uses `parallel` in the last line but that was never imported.

As per [the documentation](https://microsoft.github.io/just/docs/composition), the import needs to be changed to:

```js
const { task, option, logger, argv, parallel } = require('just-task');
```

Tangentially related, I would suggest to replace that image with an actual code snippet. While these hero code images look fancy, they are not accessible at all.

**Comment by newyork-anthonyng** [comment]
This is a duplicate of #140 , and should be addressed by PR #154

**Comment by kenotron** [comment]
Just merged that, so I'll close this one.
